### PR TITLE
Community page

### PR DESCRIPTION
**Pull request by wisnie**

### CODE REVIEW

File: components/composited/CommunitiesPage/CommunitiesPage.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      />
+      <ul className="flex flex-col gap-4">
+        {communities.map(({ city, name }) => {
+          return (
+            <li key={name}>
+              <CommunityCard
+                name={name}
+                city={city}
+                projectsNo={42}
+                usersNo={31}
```
Comment (wisnie) [inline components/composited/CommunitiesPage/CommunitiesPage.tsx:33]: @todo fetch relations from DB.

File: components/composited/CommunityPage/CommunityPage.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    <section>
+      <Heading
+        className="mb-4"
+        as="h1"
+        variant="base"
+        content={community.name}
+        displayDecorationBorder={true}
+      />
+      {/* Description */}
+      <p className="text-xs leading-5 font-normal text-gray-700"></p>
```
Comment (wisnie) [inline components/composited/CommunityPage/CommunityPage.tsx]: Modify data model to include description and add slug field.

File: components/composited/CommunityPage/CommunityPage.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import { Community } from '../../../validators/Community';
```
Comment (wisnie) [inline components/composited/CommunityPage/CommunityPage.tsx:4]: Modify `Community` schema to validate against `usersNo` and `projectsNo`.

File: pages/_app.tsx [hunk 4]
```diff
@@ -1,6 +1,7 @@
 import { useState } from 'react';
 import type { AppProps } from 'next/app';
 
+import { ReactQueryDevtools } from '@tanstack/react-query-devtools';
```
Comment (wisnie) [inline pages/_app.tsx]: Remove for production

**Review by Skczu — COMMENTED** [review]
App doesn't build, isn't that something to fix before merging?

**Pull request merged**